styles.css

```diff
@@ -0,0 +1,113 @@
+* {
+  box-sizing: border-box;
+}
+
+#navbar {
+  display: flex;
+  justify-content: space-between;
+  margin-left: 0;
+  height: 55px;
+  width: 375px;
+  background-color: #3c3a39;
+}
+
+#bars {
+  margin-top: 15px;
+}
+
+.bar1,
+.bar2,
+.bar3 {
+  width: 20px;
+  height: 3px;
+  margin: 3px 0;
+  background-color: white;
+  margin-right: 20px;
+}
+
+#profileName {
+  color: white;
+  margin-left: 20px;
+  font-size: 18px;
+}
+
+#greetText {
+  position: static;
+  width: 327px;
+  height: 124px;
+  left: 24px;
+  top: 120px;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 25px;
+  line-height: 50px;
+  letter-spacing: 0.37px;
+  color: #ff6b00;
+}
+
+#container {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 120px 24px;
+  position: absolute;
+  width: 375px;
+  height: 590px;
+  letter-spacing: 0;
+  top: 55px;
+  background: #1c1a19;
+}
+
+#jobTitle {
```

Comment:
- [ ] It's always a best practice to always use the class attribute for multiple elements (do not use ID selector  for multiple elements), kindly change your id into class selecteur you can visit this [link](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) for more information